### PR TITLE
Link to the JARs on Mvn Central

### PR DESCRIPTION
**Pull request by holyjak**
because I could not find them easily

### CODE REVIEW

File: opentracing-shim/README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 [![Javadocs][javadoc-image]][javadoc-url]
-# OpenTelemetry - OpenTracing Shim
+# OpenTelemetry - OpenTracing Shim (Bridge)
```
Comment (jkwatson) [inline opentracing-shim/README.md:2]: I'm not totally opposed to adding the parenthetical, but it's not the naming that we've been using on the project. Do you think this makes things more clear or discoverable?

Comment (anuraaga) [reply opentracing-shim/README.md:2]: Yeah agree with @jkwatson - Shim is already a bridge so it seems like stuttering, I'd leave it out.

Comment (jkwatson) [inline opentracing-shim/README.md:2]:
```suggestion
# OpenTelemetry - OpenTracing Shim
```

**Comment by jkwatson** [comment]
@holyjak thanks for the contribution. Looks like there are a couple of comments to take a look at, and you'll need to sign the CLA before we can get it merged. Thanks!

File: opentracing-shim/README.md (hunk 1 above)
Comment (breedx-splk) [reply opentracing-shim/README.md:2]: Also worth pointing out that "bridge" has a different meaning in the instrumentation/agent project, so I'd avoid that here.

**Comment by jkwatson** [comment]
closing this, because we cannot merge without the CLA being signed. I created an issue to update our docs before release: #2393

**Pull request closed without merging**